ECHO-COVID19.ipynb

```diff
@@ -46,8 +65,195 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Select your state\n",
-    "By typing in its postal code abbreviation"
+    "## National Trends"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Air emissions\n",
+    "First, are facilities even doing stack tests?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql = \"select * from `ICIS-AIR_STACK_TESTS` where ACTUAL_END_DATE like '04/%'\"\n",
+    "url='http://apps.tlt.stonybrook.edu/echoepa/?query='\n",
+    "data_location=url+urllib.parse.quote(sql)\n",
+    "\n",
+    "data = pd.read_csv(data_location,encoding='iso-8859-1',header = 0)\n",
+    "data.set_index(\"pgm_sys_id\", inplace=True)\n",
+    "data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data['ACTUAL_END_DATE'] = pd.to_datetime(data['ACTUAL_END_DATE'], format=\"%m/%d/%Y\") #format\n",
+    "\n",
+    "stacks = data.groupby(['ACTUAL_END_DATE'])['STATE_EPA_FLAG'].count() \n",
+    "# = total number of pollutants stack test. May want to summarize by facility instead. \n",
+    "stacks = stacks.resample('M').sum() #resample to a monthly basis  \n",
+    "stacks = stacks.loc[(stacks.index.month == 4)] # Filter back to just Aprils\n",
+    "stacks = pd.DataFrame(stacks)\n",
+    "\n",
+    "ax = stacks.plot(kind='bar', title = \"Past Aprils - # of pollutants stack-tested\", figsize=(20, 10), fontsize=16)\n",
+    "ax\n",
+    "\n",
+    "#label trendline\n",
+    "trend=stacks['STATE_EPA_FLAG'].mean()\n",
+    "ax.axhline(y=trend, color='r', linestyle='-')\n",
+    "\n",
+    "#label past 4 Aprils trend (2017, 2018, 2019, 2020)\n",
+    "trend_month=pd.concat([stacks.loc[\"2017-04\"],stacks.loc[\"2018-04\"],stacks.loc[\"2019-04\"]])\n",
+    "trend_month=trend_month['STATE_EPA_FLAG'].mean()\n",
```

Comment:
As a non-expert I don't really know what to take away from these two cells. Why are we looking at Aprils? Can these timestamps be made more readable? Can the Y-axis be labeled? Can the title be bigger? What is ACTUAL_END_DATE? What do the yellow and red lines mean?

Reply:
Hmm. Elsewhere there is some commentary about making an apples-apples comparison because of some pollutant releases may be seasonal. Will try to make clearer. 

The red/yellow lines are described in the text, but could be in the legend.

I am not sure about making the title bigger!

Re `Actual_End_date` - yes, need to change dataset column names to something meaningful...

Reply:
Add to #11, #12